applications/nrf_desktop/README.rst

```diff
@@ -25,24 +35,78 @@ Supported boards:
 * PCA20045 - nRF Desktop casual mouse reference design (nrf52810)
 * PCA10059 - nRF Desktop dongle reference design
 
+The application was designed to allow easy porting to a new HW. Please check
+DUPA to get information about the device porting steps.
+
+
+Dependencies
+************
+
+This application uses the following |NCS| libraries and drivers:
+
+    * :ref:`event_manager`
+    * :ref:`hids_readme`
+    * :ref:`hids_c_readme`
+    * :ref:`nrf_bt_scan_readme`
+    * :ref:`gatt_dm_readme`
+    * ``drivers/sensor/paw3212``
+    * ``drivers/sensor/pmw3360``
+
+
 Building and Running
 ********************
+
 .. |sample path| replace:: :file:`applications/nrf_desktop`
 
+The nRF Desktop application is build in the same way to any other NCS
+application or sample.
 .. include:: /includes/build_and_run.txt
 
+There can be several configuration files dedicated for each board. They
```

Comment:
Would this sentence and the following one sound more appropriate in the section about supported boards?

Reply:
The context is different - we need to tell which of various possible variants is chosen for building. When board is added it should contain at least one configuration  (for one of the build variants).

Reply:
I will rephrase it so maybe meaning will be easier to understand.